app/client/src/components/ads/TreeDropdown.tsx

```diff
@@ -53,18 +53,25 @@ export type TreeDropdownProps = {
   modifiers?: IPopoverSharedProps["modifiers"];
   onMenuToggle?: (isOpen: boolean) => void;
   position?: Position;
+  menuWidth?: number;
+};
+
+export type StyledMenuProps = {
+  width?: number;
 };
 
 export const StyledMenu = styled(Menu)`
   max-height: ${(props) =>
     `calc(100vh - ${props.theme.smallHeaderHeight} - ${props.theme.bottomBarHeight})`};
   overflow: auto;
-  min-width: 260px;
+  min-width: 220px;
+  width: ${(props: StyledMenuProps) => `${props.width}px`};
   padding: 0px;
   border-radius: 0px;
   background-color: ${(props) => props.theme.colors.treeDropdown.menuBg.normal};
   .${Classes.MENU} {
-    min-width: 260px;
+    min-width: 220px;
```

Comment:
```suggestion
    min-width: 260px;
```
We can undo min-widget to 260px correct?